src/utils/image-list.ts

```diff
@@ -9,13 +9,15 @@ export const images = {
   checkboxEmpty: '/checkbox-empty.svg',
   checkboxFilled: '/checkbox-filled.svg',
   checkBlack: '/check-black.svg',
-  checkGreen: '/check-green.svg',
+  checkCircle: '/check-circle.svg',
+  checkCircleFill: '/check-circle-fill.svg',
   close: '/close.svg',
-  closePink: '/close-pink.svg',
   connected: '/connected.svg',
   connectedDark: '/connected-dark.svg',
   disconnect: '/disconnect.svg',
   error: '/error.svg',
+  errorCircle: '/error-circle.svg',
```

Comment:
I'm not so sure we should be continuing with this approach (of adding these SVGs as public images). Have you considered taking the approach that @Anboias has been using (i.e. adding them as icon components)?

Reply:
My approach is to use react components only when we use the icons in multiple places in multiple colors or styles. I am not sure I can see a point in making every image a react component. But I don't feel strongly about this.

Reply:
@mcoetzee After @Anboias merged his PRs there is a `check-circle-fill` icon so I will probably follow the approach and make icon also from `check-circle`, `error-circle` and `error-circle-fill`. However, I would like to do it in https://github.com/api3dao/api3-dao-dashboard/pull/502 which is based on this PR to avoid conflicts. Are you ok with that?

Reply:
Sounds good 👍